projects/aca-shared/src/lib/components/toolbar/toolbar.component.html

```diff
@@ -1,5 +1,5 @@
 <mat-toolbar>
   <ng-container *ngFor="let item of items; trackBy: trackByActionId">
-    <aca-toolbar-action [actionRef]="item"></aca-toolbar-action>
+    <aca-toolbar-action [ngClass]="{'rotate-icon': item.icon === 'unfold_more'}" [actionRef]="item"></aca-toolbar-action>
```

Comment:
this is incorrect, you are hacking all actions to have static icon

Reply:
if there's more than one icon here, are you going to hardcode dozens of icon names?

Reply:
solution: just register the svg image with the icon pointing to the right direcion

Reply:
Implemnted the suggested solutiuon.